libs/langchain/langchain/graphs/falkordb_graph.py

```diff
@@ -49,12 +57,29 @@ def get_schema(self) -> str:
         """Returns the schema of the FalkorDB database"""
         return self.schema
 
+    @property
+    def get_structured_schema(self) -> Dict[str, Any]:
+        """Returns the structured schema of the Graph"""
+        return self.structured_schema
+
     def refresh_schema(self) -> None:
         """Refreshes the schema of the FalkorDB database"""
+        node_properties = self.query(node_properties_query)
+        rel_properties = self.query(rel_properties_query)
+        relationships = self.query(rel_query)
+
+        self.structured_schema = {
+            "node_props": {
+                el[0]["labels"]: el[0]["properties"] for el in node_properties
```

Comment:
seems node_properties is a list of dicts, in which case `el` is a dict?

Reply:
let me first rebase and merge this PR, after the merge of this PR https://github.com/langchain-ai/langchain/pull/11122

Reply:
@baskaryan OK, it's ready for review